test/test_ops.py

```diff
@@ -1258,6 +1258,83 @@ def test_giou_jit(self) -> None:
         self._run_jit_test([[0, 0, 100, 100], [0, 0, 50, 50], [200, 200, 300, 300]])
 
 
+class TestDistanceBoxIoU(BoxTestBase):
+    def _target_fn(self):
+        return (True, ops.distance_box_iou)
+
+    def _generate_int_input():
```

Comment:
This has nothing to do with this PR, but I want make this visible. Although this works, there are two issues with this structure introduced in #5380 cc @datumbox:

1. We shouldn't have regular methods without `self` as first parameter. Given that we don't need it here, we should use the `@staticmethod` decorator.
2. It makes little sense to separate the input and expected values into two methods given that they are always used together. This just makes it harder to read.

Reply:
+1 @yassineAlouini , Even I was thinking about this.

Reply:
> I guess we should keep this for another PR?

:+1:

Reply:
Is the suggested plan to fix this on a separate PR across all tests? Just trying to figure out if this is blocking of this PR and whether we need to take similar action at #5776. @pmeier it's your call.

Reply:
No this is not blocking for this PR. I would appreciate if either @yassineAlouini or  @abhi-glitchhg could send a follow-up PR fixing this. Whoever wants to pick this up, please comment here to avoid duplicate work and resolve this comment afterwards.

Reply:
Hey @pmeier, sorry :(  
I have uni exams starting next week, so I will not be able to devote much time to this.

Reply:
No worries @abhi-glitchhg. Good luck with that. @yassineAlouini would you be able to pick this up? Otherwise, I'll open an issue so we don't forget and we can do this later.

Reply:
@pmeier Yes, I have time this afternoon so will take this on the next PR. :ok_hand:

Reply:
And good luck @abhi-glitchhg in your exams. :+1: